packages/components/src/DateTimePickers/generator.js

```diff
@@ -35,17 +35,20 @@ export function buildDayNames(firstDayOfweek = 1, t) {
 /**
  * Generate the set of weeks for a specific month
  */
-export function buildWeeks(year, monthIndex, firstDayOfWeek = 1) {
+export function buildWeeks(year, monthIndex, firstDayOfWeek = 1, show6weeks = false) {
```

Comment:
I don't think this is useful : `show6weeks`
The only UI is with 6 weeks

Reply:
but with the parameter, user could show different view if he like does not like to show 6 weeks.

Reply:
yes, you are right. will update it.

Reply:
show6weeks is removed